dev/conformance/runner.ts

```diff
@@ -248,7 +250,8 @@ function commitHandler(spec) {
     try {
       assert.deepEqual(request, convertProto.commitRequest(spec.request));
 
-      const res = {
+      // tslint:disable-next-line:no-any
+      const res: any = {
```

Comment:
This is https://github.com/googleapis/nodejs-firestore/blob/master/dev/protos/firestore_proto_api.d.ts#L2035